src/mainboard/protectli/vault_adl_p/Kconfig

```diff
@@ -0,0 +1,84 @@
+if BOARD_PROTECTLI_VP66XX
```

Comment:
Just a comment: technically, the line will be VP66X0, but since we already have 46XX we can follow this way.